### PR TITLE
fix: add separate issue template

### LINKED ISSUE #2418: Add separate issue templates
**Actual Behaviour**

There are no separate issue templates

**Expected Behaviour**

Github now supports separate issue templates for various occasions.Split the current issue template into 3 parts - bug, feature, and chore

Add 3 corresponding buttons to README in create an issue section where a user can be redirected to new issue template respectively

similar to what has been done in the open-event-orga-app

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by nikit19**
Fixes #2418 

Changes: Separate issue templates can now be used from the README.md

Screenshots for the change: N/A

### CODE REVIEW

File: ISSUE_TEMPLATE/chore.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Chore
+
+**Description**
+
+Please describe the tools/files you want to update/upgrade.
```
Comment (iamareebjamal) [inline ISSUE_TEMPLATE/chore.md]: Please describe the task ...

Comment (nikit19) [reply ISSUE_TEMPLATE/chore.md]: PR updated

File: ISSUE_TEMPLATE/chore.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Chore
+
+**Description**
+
+Please describe the task you want to update/upgrade.
```
Comment (iamareebjamal) [inline ISSUE_TEMPLATE/chore.md]: `task you want to update/upgrade.`

We don't upgrade tasks

Comment (iamareebjamal) [reply ISSUE_TEMPLATE/chore.md]: chore is not always about upgrading

Comment (nikit19) [reply ISSUE_TEMPLATE/chore.md]: PR updated

File: ISSUE_TEMPLATE/chore.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+### Chore
+
+**Description**
+
+Please describe the task you want to update.
```
Comment (iamareebjamal) [inline ISSUE_TEMPLATE/chore.md]: task is also not about updating. Please extrapolate from reviews :)

Comment (nikit19) [reply ISSUE_TEMPLATE/chore.md]: I am really sorry for the confusion.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**